ckanext/data_qld/templates/emails/bodies/new_datarequest_organisation.txt

```diff
@@ -1,5 +1,9 @@
 A new data request has been added and assigned to your organisation.
 
+To action or 'Reply' to this data request, follow this link:
 {{ datarequest['dataset_url'] }}
 
+Data request subject: {{ datarequest['title'] }}
```

Comment:
Hi @MarkCalvert 
Looks good - could we try escaping the datarequest title and description in the email template using the `|e` filter in Jinja2 (ref.: https://jinja.palletsprojects.com/en/2.11.x/templates/#working-with-manual-escaping) - just to be safe?
Thanks.

Reply:
Hi @salsa-nathan,
Good spotting, I have added `striptags` to remove all HTML tags.
Thanks